src/JSInterop/Microsoft.JSInterop/test/Infrastructure/DotNetDispatcherTest.cs

```diff
@@ -878,6 +917,26 @@ public async Task<InvokableAsyncMethodResult> InvokableAsyncMethod(TestDTO dtoVi
             };
         }
 
+        [JSInvokable]
+        public async ValueTask<InvokableAsyncMethodResult> InvokableAsyncMethodReturningValueTask(TestDTO dtoViaJson, DotNetObjectReference<TestDTO> dtoByRefWrapper)
+        {
+            await Task.Delay(50);
```

Comment:
There's no need for Task.Delay, we don't do this in our tests as it artificially slows runs